specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,670 @@
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Security/integrations": {
+      "get": {
+        "x-ms-examples": {
+          "List integrations by subscription scope": {
+            "$ref": "./examples/integrations/ListBySubscriptionIntegrations_example.json"
+          },
+          "List integrations by security connector scope": {
+            "$ref": "./examples/integrations/ListBySecurityConnectorIntegrations_example.json"
+          },
+          "List integrations by management group scope": {
+            "$ref": "./examples/integrations/ListByManagementGroupIntegrations_example.json"
+          }
+        },
+        "tags": [
+          "Integrations"
+        ],
+        "description": "Get a list of all relevant integrations over a scope",
+        "operationId": "Integrations_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/getIntegrationTablesConfigurations": {
+      "post": {
+        "x-ms-examples": {
+          "Try to get Service Now tables configurations by subscription scope": {
+            "$ref": "./examples/integrations/TryGetServiceNowTablesConfigurationsBySubscription_example.json"
+          },
+          "Try to get Service Now tables configurations by security connector scope": {
+            "$ref": "./examples/integrations/TryGetServiceNowTablesConfigurationsBySecurityConnector_example.json"
+          },
+          "Try to get Service Now tables configurations by management group scope": {
+            "$ref": "./examples/integrations/TryGetServiceNowTablesConfigurationsByManagementGroup_example.json"
+          }
+        },
+        "tags": [
+          "TryGetServiceNowTablesConfigurations"
+        ],
+        "description": "Try to get Service Now tables configurations over a scope",
+        "operationId": "ServiceNowTablesConfigurations_Get",
```

Comment:
operationId are used for generating proper SDK models and namespaces - if it is strictly used for 'integrations', then perhaps a more suitable name for the operation would be `Integrations_GetIntegrationTablesConfigurations`